project.clj

```diff
@@ -8,15 +8,12 @@
 
   :min-lein-version "2.7.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "0.6.1"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.8.1-SNAPSHOT"]
                    :inherit [:managed-dependencies]}
 
   :dependencies [[puppetlabs/pcp-common "1.1.4"]
 
-                 ;; Transitive dependencies on jetty for stylefuits/gniazdo
-                 ;; to use a stable jetty release (gniazdo specifies 9.3.0M1)
-                 [org.eclipse.jetty.websocket/websocket-client "9.2.10.v20150310"]
```

Comment:
oh that was because I still had the parent that specified if I roll back the parent I get

Provided artifact is missing a version: [org.eclipse.jetty.websocket/websocket-client nil]